index.js

```diff
@@ -3,18 +3,23 @@
 	Author Tobias Koppers @sokra
 */
 var path = require("path");
-
 var loaderUtils = require("loader-utils");
 var validateOptions = require('schema-utils');
+var schema = require('./options.json')
+
+var validOptions = new WeakSet();
 
 module.exports = function () {};
 
 module.exports.pitch = function (request) {
 	if (this.cacheable) this.cacheable();
 
-	var options = loaderUtils.getOptions(this) || {};
-
-	validateOptions(require('./options.json'), options, 'Style Loader')
+  var options = loaderUtils.getOptions(this);
```

Comment:
Avoiding `|| {}` can break some places, it is good practice use `options.myOption`

Reply:
This (I believe) matches behavior prior to my change. Since an empty options `{}` is subsequently populated (aka when expected props are `typeof options.x === 'undefined'`) - we can safely assume that our internal defaults are valid.

Reply:
:+1: